### PR TITLE
Bug 1418364 Replaced Version with <tag> under Deploying the EFK Stack

### PR DESCRIPTION
**Pull request by mburke5678**
I replaced the hard-coded version number with the <tag> variable and added the variable value to the header, with separate variable/value sets for Origin and Enterprise.

### CODE REVIEW

**Comment by mburke5678** [comment]
@adellape You helped with this bug last week; you figured out how to use ifdef's in the header, if you recall. Can you take a quick look to make sure this PR is good to go?

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -336,17 +342,21 @@ With parameters:
...
 endif::openshift-origin[]
 ifdef::openshift-enterprise[]
 ----
 $ oc new-app logging-deployer-template \
-             --param IMAGE_VERSION=3.3.0 \
+             --param IMAGE_VERSION=<tag> \
              --param MODE=install
 ----
+
 endif::openshift-enterprise[]
```
Comment (adellape) [inline install_config/aggregate_logging.adoc]: Now that's there's no actual difference between these code blocks (thanks to the use of `<tag>`), you can remove the ifdefs here and just have a single code block / command, I think.

**Comment by mburke5678** [comment]
@adellape Removing the second ifdef section and going with one code block with no ifdef seems to have worked in _preview

**Comment by mburke5678** [comment]
[rev_history]
|xref:../install_config/aggregate_logging.adocc#install_config/aggregate_logging.adoc[Aggregating Container Logs]
|Added a version variable and <tag> to code block example in xref:../install_config/aggregate_logging.adoc#deploying-the-efk-stack[Pre-deployment Configuration] section to display the correct current version to use.
%

**Comment by vikram-redhat** [comment]
@mburke5678 - I have added it to the release, but make sure that commits are squashed. There should be only one commit per PR. @adellape .

**Pull request merged**